src/data/ProgramsD2Repository.ts

```diff
@@ -87,20 +101,109 @@ export class ProgramsD2Repository implements ProgramsRepository {
         return d2ProgramRules.run(options);
     }
 
+    async getAll(options: MoveProgramAttributeOptions): Async<ProgramAttributes[]> {
+        const trackedEntities = await this.getFromTracker<D2TrackedEntity>("trackedEntities", {
```

Comment:
We were using manual calls to the new tracker, but at this point d2-api already should support see (https://github.com/EyeSeeTea/d2-api/pull/135)  from v1.14.0. Check with @adrianq if we do this change now or we'll start using it on future developments.

Reply:
I tried to update but there was some errors in `ProgramEventsD2Repository`. @adrianq told me we can postpone the migration since we will need to do some test in other scripts not related with the current one.